secrets.test.js

```diff
@@ -13,9 +13,8 @@ test('export dynamic secrets', async () => {
         'secret_access_key': 'aws-secret-key',
         'session_token': 'aws-session-token',
     }
-
+    core.setOutput = jest.fn(() => {});
     core.exportVariable = jest.fn(() => {});
-    core.setSecret = jest.fn(() => {});
```

Comment:
Related to the comment below (I went out of order): since `core.setOutput` is being called and `core.setSecret` is being called, and they are both important, let's just test that both are called.